meta-resin-common/recipes-support/hostapp-update-hooks/files/99-resin-grub

```diff
@@ -6,17 +6,52 @@
 
 set -o errexit
 
+verifyMd5sum() {
+  if [ "$1" = "$3" ] ; then return 1; fi
+}
+
+getSize() {
+  echo $(du -sb "$1" | awk '{print $1}')
+}
+
 DURING_UPDATE=${DURING_UPDATE:-0}
 
 if [ "$DURING_UPDATE" = "1" ]; then
-	SYSROOT="/mnt/sysroot/inactive"
+    SYSROOT="/mnt/sysroot/inactive"
 else
-	SYSROOT="/mnt/sysroot/active"
+    SYSROOT="/mnt/sysroot/active"
 fi
 
 new_part=$(findmnt --noheadings --canonicalize --output SOURCE $SYSROOT)
 new_part_label=$(blkid "$new_part" | awk '{print $2}' | cut -d'"' -f 2)
 
+# flash legacy grub only if we do not support UEFI
+if [ ! -d /sys/firmware/efi ] ; then
+    # Remove EFI
+    rm -rf "$RESIN_BOOT_MOUNTPOINT/EFI" || true
```

Comment:
The hook doesn't work as RESIN_BOOT_MOUNTPOINT is unavailable.

Need to run `. /usr/sbin/resin-vars`  for the env var to exist.